src/controllers/dlgprefcontroller.cpp

```diff
@@ -550,8 +551,21 @@ void DlgPrefController::slotApply() {
     }
 
     QString mappingPath = mappingPathFromIndex(m_ui.comboBoxMapping->currentIndex());
+
+    QFileInfo mappingFileInfo(mappingPath);
```

Comment:
```suggestion
    auto mappingFileInfo = QFileInfo(mappingPath);
```
We prefer this, because the assignment of a new local object is emphasized.